openapi.yaml

```diff
@@ -4969,6 +4981,8 @@ paths:
                     by using the [GET /regions](/api/v4/regions) endpoint.
                     A cross-region migration will cancel a pending migration
                     that has not yet been initiated.
+
+                    A cross-region migration will initiate a `linode_migrate_datacenter_create` event.
```

Comment:
We're using "cross data center" instead of cross-region above. Either this or above should change to be consistent.

Reply:
Good catch. Updated!